pkg/config/user_config.go

```diff
@@ -510,6 +511,7 @@ func GetDefaultConfig() *UserConfig {
 				MoveDownCommit:                 "<c-j>",
 				MoveUpCommit:                   "<c-k>",
 				AmendToCommit:                  "A",
+				ResetCommitAuthor:              "a",
```

Comment:
I can't think of any better option than this keybinding. Although 'a' typically means 'select everything in this panel', it doesn't make any sense to select all commits. And I see the value in having 'A' and 'a' both relating to amending, with the former being for amending file changes, and the latter being for amending metadata like the author.

For what it's worth in future I want this keybinding to open a menu with an additional option to edit the author explicitly, but we don't need to do that in this PR.